src/test/resources/application-connected-test.yml

```diff
@@ -0,0 +1,3 @@
+workspace:
```

Comment:
How do you see this file evolving? I'm wondering how it's going to include `folder-id` values for other environments, in addition to that of dev.

Reply:
We don't need to run these tests against other environments. Just as we don't run unit tests "against" environments, we don't run connected tests against environments. Rather, we run the tests within a deployment environment where we have access to the dependencies and resources we need. For the test I'm adding here, we need a Service Account that has the privileges to create projects within a folder. There is such a credential and folder within the "dev" environment, and our `render_config.sh` already grabs dev SA credentials.
We don't  even necessarily need to run against "dev", we could run against a test only environment, or preview environments. Dev's just convenient at the moment as preview environments aren't ready.

I expect us to add more configuration values as needed to this file. In an ideal world, more of this test configuration that's based on the output of terraform would be rendered somewhere central, both for testing and for different helm deployments of environments. We're not here yet - we don't have a good way to pipe values from terraform to k8s helm/test config. So, for now, we're hard coding this value and hard coding the vault location of the service account credentials.
Happy to talk more offline too.